### ISSUE TITLE
Update OKD-based CRC build

### ISSUE THREAD

**Issue by ddsharpe** [openshift/okd#636]
Failed to renew TLS certificates: please check if a newer CodeReady Containers release is available: Temporary error: certificate /var/lib/kubelet/pki/kubelet-server-current.pem still expired (x60)

When retrieving the latest CRC from okd.io/crc.html (at https://dl.fedoraproject.org/pub/alt/okd-crc/linux-amd64/ ), the crc is dated 2020-12-15 and has outdated certificates.

**Comment by rjeberhard** [comment]
Just following up... It looks like @vrutkovs tried to assign @cgruver, but that it didn't go through.

**Comment by vrutkovs** [comment]
Hmm, right, in any case Charro should have received a notification and hopefully would take a look.

Perhaps we should rename this into "Update OKD-based CRC build" and track progress there

**Comment by rjeberhard** [comment]
Thanks, @vrutkovs, I appreciate it. We have a customer who doesn't want to use the click-through license for other OKD / OpenShift installation options and so is presently blocked because using CRC seems to be the only fully open option (unless I'm missing another choice).

**Comment by holzman** [comment]
Any updates? This is an issue for us using CRC too.

**Comment by byako** [comment]
Is this issue related to https://www.okd.io/crc.html not showing any releases for download, or should I file new issue ?

**Comment by cgruver** [comment]
Hey all.  Apologies for the late response.  I am blocked by an issue building a 4.7 release, but believe that I have a work around.

I'll post an update.

**Comment by cgruver** [comment]
Coming soon.  Hopefully today.

I got a successful build done yesterday, but still had to do a couple of things manually to get CRC to run.

I've added those steps to the build and will try again today after #dayjob.

**Comment by cgruver** [comment]
I've got a build running right now which I hope to be able to release today.

**Comment by cgruver** [comment]
Try the files uploaded here:

https://dl.fedoraproject.org/pub/alt/okd-crc/release/

OKD.io needs to be fixed, but these binaries should work.

**Comment by cgruver** [comment]
OKD.io is showing the correct downloads now.

@vrutkovs  This can be closed.

**Comment by vrutkovs** [comment]
Perfect, thank you!
